dask_expr/io/io.py

```diff
@@ -147,6 +154,58 @@ def _tune_up(self, parent):
         return
 
 
+class FusedParquetIO(FusedIO):
+    _parameters = ["_expr"]
+
+    @functools.cached_property
+    def _name(self):
+        return (
+            funcname(type(self.operand("_expr"))).lower()
+            + "-fused-parq-"
+            + _tokenize_deterministic(*self.operands)
+        )
+
+    @staticmethod
+    def _load_multiple_files(frag_filters, columns, schema, index_name):
+        from dask_expr.io.parquet import ReadParquetPyarrowFS
+
+        # Note: Ideally we'd build a pyarrow fragment that points to multiple
+        # tables since pyarrow handles IO and CPU threading separately and we're
+        # just overcomitting threads here. However, this isn't exposed at time
+        # of writing
+        # TODO: We may want to use a gloal pool with sufficient threads since
+        # executing this task in a local executor could be a little too much
+        tables = (
+            ReadParquetPyarrowFS._fragment_to_table(
+                frag,
+                filter,
+                columns,
+                schema,
+            )
+            for frag, filter in frag_filters
+        )
```

Comment:
In a future where we tried using python threads for download concurrency, this is where we would use a ThreadPoolExecutor?  Is that correct?

Reply:
yes